src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,6 +263,19 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
+        /** 
+         * Represents the 'FuzzResultTableEntry' as a string with all values separated by a comma. Is called by
+         * the Export button's actionListener in 'HttpFuzzResultsContentPanel'. 
+         * 
+         * A HttpFuzzerReflectionDetectorStateHighlighter is used to reveal true value of the customState field
+         */
+        public String toString() {

```

Comment:
Not all fields, just the ones that are shown in the results. Some of the columns are hidden by default, but should be included anyway.
Note that there's a lot of corner cases to handle when converting to CSV, that's why I was suggesting the use of the library.